tests/profiles/stat_test.py

```diff
@@ -144,6 +144,19 @@ def test_check_italic_axis_in_stat():
     assert_results_contain(check(fonts),
                            FAIL, "missing-roman")
 
+    fonts = [
+        TEST_FILE("shantell/ShantellSans[BNCE,INFM,SPAC,wght].ttf"),
+        TEST_FILE("shantell/ShantellSans-Italic[BNCE,INFM,SPAC,wght].ttf"),
+    ]
+    # Remove ital axes
+    for i, font in enumerate(fonts):
+        ttFont = TTFont(font)
+        # These two fonts actually have the ital axis in different positions
+        if i == 0:
+            ttFont["STAT"].table.DesignAxisRecord.Axis = [ttFont["STAT"].table.DesignAxisRecord.Axis[0]] + ttFont["STAT"].table.DesignAxisRecord.Axis[2:]        
+        elif i == 1:
+            ttFont["STAT"].table.DesignAxisRecord.Axis = ttFont["STAT"].table.DesignAxisRecord.Axis[:-1]        
+        ttFont.save(font.replace(".ttf", ".missingitalaxis.ttf"))
```

Comment:
This is a bit more complicated than I expected. But it seems to work.
I may be able to simplify this code. I may give it a try after merging this PR.

Reply:
Wait. I should go the extra mile and repeat the same process of moving the ital axis for the upright font as well. It's laziness, that's all

Reply:
Done